superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterDivider.tsx

```diff
@@ -0,0 +1,191 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { css, SupersetTheme } from '@superset-ui/core';
+import React, { useEffect, useRef, useState } from 'react';
+import Icons from 'src/components/Icons';
+import { Tooltip } from 'src/components/Tooltip';
+
+export interface FilterDividerProps {
+  title: string;
+  description: string;
+  horizontal?: boolean;
+  horizontalOverflow?: boolean;
```

Comment:
I'm curious whether the `verticalOverflow`  might be in the future. if it is, should we use props like `overflow` to control this?

Reply:
Sounds good, changed `horizontalOverflow` to `overflow` in case we want to support that in the future.  Since the styles are unique, another option is that I could have an `orientation` field that accepts constants for each supported orientation (vertical, horizontal, horizontal overflow) and we could add a 4th constant if we end up wanting to support vertical overflow dropdown – let me know if that sounds better.